### DISCUSSION TITLE
loading force from DFPT

### DISCUSSION THREAD

**Discussion by whzhangg** [ttadano/alamode#22] [Q&A] [answered]
Dear Tadano
Is it possible to load the DFPT force constant to the alamode code by manually preparing the .xml file?

**Comment by ttadano** [comment, marked as answer]
You can convert a Quantum ESPRESSO .fc file to ALAMODE .xml file by using ``qe2alm.cpp`` in the ``tools`` directory.

1. Edit ``Makefile`` in the ``tools`` directory.
2. Create ``qe2alm`` binary via ``make``.
3. Run ``qe2alm`` in the foreground. The program will ask you the filenames of input fc file and output xml file.
4. Load the created .xml file in ``anphon``.

**Comment by whzhangg** [comment]
I will try. I didn't notice this file before, Thank you.
